modules/RTC/RTCUtils.js

```diff
@@ -779,10 +360,17 @@ class RTCUtils extends Listenable {
             };
             this.getTrackID = ({ id }) => id;
         } else {
```

Comment:
Do we want to add a check here for `browser.isSupported()` ? Don't we want to throw the endpoint not webrtc-capable error if its not supported ?